demo/README.md

```diff
@@ -1,174 +1 @@
-  --node-socket devnet/node.socket
-```
-
-Carol's node:
-```sh
-cabal exec hydra-node -- \
-  --node-id 3 --port 5003 --api-port 4003 --monitoring-port 6003 \
-  --peer localhost:5001 \
-  --peer localhost:5002 \
-  --hydra-signing-key carol.sk \
-  --hydra-verification-key alice.vk \
-  --hydra-verification-key bob.vk \
-  --cardano-signing-key devnet/credentials/carol.sk \
-  --cardano-verification-key devnet/credentials/alice.vk \
-  --cardano-verification-key devnet/credentials/bob.vk \
-  --ledger-genesis devnet/genesis-shelley.json \
-  --ledger-protocol-parameters devnet/protocol-parameters.json \
-  --network-id 42 \
-  --node-socket devnet/node.socket
-```
-
-If things go well, the nodes should start logging once they are connected to the chain.
-
-
-# Seeding Network
-
-In the current stage of development, Hydra nodes need a specially crafted set of UTXO to drive the Head protocol ("fuel") and of course some UTXO to be committed to the head.
-
-## With Docker
-
-The included script `seed-devnet.sh` uses the `cardano-cli` in the already running `cardano-node` container to give Alice, Bob and Carol some UTXO to commit and some "fuel" UTXO.
-
-NOTE: There is nothing special about those transactions so one could any other Cardano client to create those transactions. This transaction must match the following characteristics:
-* It must pay outputs to commit to the key that's used by the Hydra Node's internal wallet, as defined by argument `--cardano-signing-key` of `hydra-node` executable,
-* One of the outputs must include datum hash `a654fb60d21c1fed48db2c320aa6df9737ec0204c0ba53b9b94a09fb40e757f3` as this is our "fuel" marker.
-
-## Without Docker
-
-You can use the `seed-devnet.sh` script by passing it the path to a `cardano-cli` executable to use instead of having it using the docker container, e.g.
-
-``` sh
-./seed-devnet.sh $(which cardano-cli)
-```
-
-# Running clients
-
-## With Docker
-
-Using [`docker-compose`](https://docs.docker.com/compose/) you can start the demo _Terminal-based User Interface_ aka. `hydra-tui` to interact with Hydra nodes.
-
-There are 3 preconfigured TUIs to be run using `docker-compose`, `hydra-tui-1`, `hydra-tui-2` and `hydra-tui-3`.
-To connect to the first Hydra node in a terminal, run the following commands:
-
-``` sh
-cd demo
-docker-compose --profile tui run hydra-tui-1
-```
-
-## Without docker
-
-Connect to the nodes using `hydra-tui`. For example to use Alice's `hydra-node` and her on-chain
-credentials:
-
-```sh
-cabal exec hydra-tui -- \
-  --connect localhost:4001 \
-  --cardano-signing-key devnet/credentials/alice.sk \
-  --network-id 42 \
-  --node-socket devnet/ipc/node.socket
-```
-
-Replace port `4001` with `4002` or `4003` to connect to the other 2 nodes and
-`alice.sk` with `bob.sk` or `carol.sk` respectively.
-
-You should now be able to `[I]nit`ialise a Head, `[c]ommit` UTXOs, ...
+See the [🕮  user manual](https://input-output-hk.github.io/hydra-poc/docs/getting-started/demo).
```

Comment:
What's the rectangle glyph here? Should it be some emoji name or image?

Reply:
Yes indeed. This emoji does not appear on Mac it seems.

See why I hesitate so much for a Mac... They lack the good stuff!